dependent/population-en.csl

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-US" page-range-format="expanded" demote-non-dropping-particle="never">
```

Comment:
@nalimilan, can you remove the `class="in-text"` and `page-range-format="expanded" demote-non-dropping-particle="never"` bits here? They don't do anything in dependent styles.